packages/apiserver/src/resources/persons/person.resolver.ts

```diff
@@ -70,6 +74,21 @@ export class PersonResolver {
     }
   }
 
+  @Mutation(() => Person)
+  async updatePerson(
```

Comment:
Can you elaborate the method's name, like `updatePersonNotificationDate` or so as it only does one thing without updating other fields

Reply:
For now it's only to update updatePersonNotificationDate. However later it can be expanded.
The idea is too have generic mutation instead of adding individual mutations for updating a single field.